README.md

```diff
@@ -53,13 +53,20 @@ Optional:
  - kickstart_timezone
  - kickstart_log_host
  - dhcp_kickstart_manage_packages
- - kickstart_packages # if dhcp_kickstart_manage_packages is defined then kickstart_packages must contain %packages
+ - kickstart_packages # if dhcp_kickstart_manage_packages is defined, it must contain at least @Core or @Base
  - kickstart_extra_pre_option
  - kickstart_extra_pre_commands
  - kickstart_extra_post_commands
+ - dhcp_kickstart_centos_version set as "8" or "9" in order to skip old centos7 defaults, remember to define kickstart_packages like:
```

Comment:
FWIW this renders horribly in markdown, at least in Github markdown preview. Matters only if we strive for beauty :) Not requesting a change.